### PR TITLE
RawGit githubissues.md

### PR DESCRIPTION
**Pull request by Ebuka-Chibueze**
This is for For [Step 8](http://open-learning-exchange.github.io/#!pages/githubissues.md ) and [Issue 725](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/725) , I'm adding a picture very soon.

Raw git for reference: [here](https://rawgit.com/open-learning-exchange/open-learning-exchange.github.io/Ebuka-RawGit-Sample-1/#!pages/githubissues.md)

### CODE REVIEW

File: pages/githubissues.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ Now you can go and make the proposed changes to your local files.
...
 
-You can view your changes by going to <code>https://<b>rawgit.com</b>/YourUserName/YourUserName.github.io/YourBranchName/#!index.md</code> and viewing the page(s) you have changed. You may need to change the link slightly depending on where the file you want to view is located. If everything looks correct, you are ready to create a pull request. 
+You can view your changes by going to `https://rawgit.com/{YouGitHubUsername}/{YouGitHubUsername}.github.io/{YourBranchName}/#!index.md`and viewing the page(s) you have changed. You may need to change the link slightly depending on where the file you want to view is located. If everything looks correct, you are ready to create a pull request.
```
Comment (Ebuka-Chibueze) [inline pages/githubissues.md]: I think I need to make a separate PR for the Image upload.

Comment (sarah-lu102) [inline pages/githubissues.md]: @Ebuka-Chibueze try changing `` to `` and check in rawgit. I ran into a similar problem yesterday and that worked for me....or maybe rename the image so there is no whitespace in the image name?

Comment (mappuji) [inline pages/githubissues.md]: lets try without whitespace 😉

Comment (Ebuka-Chibueze) [inline pages/githubissues.md]: Thanks guys, I'm glad the solution was much simpler than i expected it to be.
